README.md

```diff
@@ -118,6 +118,24 @@ helm install capi-operator capi-operator/cluster-api-operator --create-namespace
 ```
 This will take some time to install and can be verified it is complete by seeing two ready pods in the `azure-infrastructure-system` namespace. 
 
+In order to install CAPI Operator with additional CRDs, you will need to use `helm install` with a `values.yaml` file and customize the `azureserviceoperator-controller-manager` deployment like so:
+
+```yaml
```

Comment:
I don't think you should put the full reference YAML example here because they don't need to modify it and it is pretty understandable how to modify it now that you have the baseline there.  A note with values file reference is good similar to what you have on line 121.  People can optionally go to the file and modify the CRDs they desire.